sdks/python/apache_beam/runners/interactive/README.md

```diff
@@ -263,6 +269,23 @@ You can choose to run Interactive Beam on Flink with the following settings.
     p = beam.Pipeline(interactive_runner.InteractiveRunner(underlying_runner=flink_runner.FlinkRunner()))
     ```
 
+*   Alternatively, if the runtime environment is configured with a Google Cloud project, you may run Flink pipelines remotely on Dataproc. To do so, only a project is required to be specified when running from an official release, otherwise, it is necessary to specify an `environment_config` option to select a specific release version. Furthermore, it is recommended to cache into a Google Cloud bucket.
```

Comment:
Maybe reword it to:

..., you can run Interactive Beam with Flink on Cloud Dataproc. To do so, configure the pipeline with a Google Cloud project. If using dev versioned Beam built from source code, it is necessary to ... option to configure a containerized Beam SDK (you can choose a released container or build one yourself).

Reply:
Sounds good! I've added your suggestion now. Thanks!